Source/Core/DolphinQt/HotkeyScheduler.cpp

```diff
@@ -169,12 +169,25 @@ void HotkeyScheduler::Run()
 
       HotkeyManagerEmu::GetStatus(true);
 
-      if (!Core::IsRunningAndStarted())
-        continue;
-
+      // Open Disc
```

Comment:
This one is a bit weirdly named. It should be Open File or Open Game. (Not sure if we have any reason to prefer either of those two, like a matching string elsewhere in DolphinQt)

Reply:
I used Open Disc because a few lines later, EjectDisc() and ChangeDisc() are grouped by the comment "Disc". Fair point though, I have changed to just Open.

Reply:
There is the important difference that Open also works with things like WAD files and DFF files, whereas eject disc and change disc actually only work with disc images.